### ISSUE TITLE
Numeric filters do not persist through refiltering/sorting

### ISSUE THREAD

**Issue by abreal-badger** [lyrisey/follicule#13]
The filters that Follicule applies for numeric values (kudos, chapters, comments, bookmarks, hits) are lost if, at some future point, the Sort and Filter button is clicked, which is a possibility if a filter is applied, the page is filtered, and then an additional filter term is added.

This appears to be an issue with how search terms are re-applied in the 'search within results' box on a new page load: manually entering a term like `kudos>1000` in the SWR box and clicking Sort and Filter results in the same behavior: the resulting search results doesn't have the term in the SWR box.

tl;dr: this is an issue with Ao3's filtering, and I don't have a good workaround for the overall problem.

**Comment by abreal-badger** [comment]
Solutions and workarounds:

Short term, apply numeric filters as the last thing you do.

Middle term, a partial interim fix might include taking the wordcount follicule and directing it to the wordcount filter boxes, as those propagate values when refiltering.

Longer term, this might integrate with issue #2 , where a workaround might involve reading the URL params and re-integrating the missing search terms into the filter UI/SWR box to allow them to propagate.
